classes/entity/SubjectDiff.php

```diff
@@ -61,10 +62,10 @@ function pull($delete = false) {
         }
 
         // check version due to 9+ requring the $project_id parameter be set without a default
```

Comment:
```suggestion
        // REDCap version 10 shuffled some functions around
```
Last nitpick, this comment is outdated and should be hijacked to reflect the new purpose